devops/scripts/clear-data.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Interesting. What is this used for?

Reply:
Its just a script I was using to reset the database while I was testing the seeding/migrations, found it easier than redeploying cockroachdb every time